.github/workflows/aks.yaml

```diff
@@ -19,11 +19,42 @@ env:
   check_url: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
 
 jobs:
+  check_changes:
+    name: Deduce required tests from code changes
```

Comment:
For reviewers: This job could also be integrated into `installation-and-connectivity` to reduce the number of different jobs displayed for each pull request. The downside of that approach is that the job would not be clearly marked as skipped (since only a subset of steps would be skipped).